.buildkite/hooks/pre-command

```diff
@@ -28,6 +29,23 @@ if [[ "$BUILDKITE_PIPELINE_SLUG" == "fleet-server" && "$BUILDKITE_STEP_KEY" == "
   export JOB_GCS_BUCKET=$(retry 5 vault kv get -field plaintext ${JOB_GCS_BUCKET_SECRET_PATH})
 fi
 
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "fleet-server-package-mbp" ]]; then
+  if [[ "$BUILDKITE_STEP_KEY" == "package-x86-64" || "$BUILDKITE_STEP_KEY" == "package-arm" || "$BUILDKITE_STEP_KEY" == "dra-snapshot" ]]; then
+    export PRIVATE_CI_GCS_CREDENTIALS_SECRET=$(retry 5 vault kv get -field plaintext ${PRIVATE_CI_GCS_CREDENTIALS_PATH})
+    export JOB_GCS_BUCKET=$(retry 5 vault kv get -field plaintext ${JOB_GCS_BUCKET_SECRET_PATH})
+  fi
+fi
+
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "fleet-server-package-mbp" && "$BUILDKITE_STEP_KEY" == "dra-snapshot" ]]; then
+    export DOCKER_USERNAME_SECRET=$(retry 5 vault kv get -field user "${DOCKER_REGISTRY_SECRET_PATH}")
+    export DOCKER_PASSWORD_SECRET=$(retry 5 vault kv get -field password "${DOCKER_REGISTRY_SECRET_PATH}")
+    docker login -u "${DOCKER_USERNAME_SECRET}" -p "${DOCKER_PASSWORD_SECRET}" "${DOCKER_REGISTRY}" 2>/dev/null
+    DRA_CREDS=$(retry 5 vault kv get -field=data -format=json "${DRA_SECRET_PATH}")
+    export VAULT_ADDR=$(echo ${DRA_CREDS} | jq -r '.vault_addr')
+    export VAULT_ROLE_ID=$(echo ${DRA_CREDS} | jq -r '.vault_role_id')
+    export VAULT_SECRET_ID=$(echo ${DRA_CREDS} | jq -r '.vault_secret_id')
```

Comment:
As there is no permission here for now, should be set some fake values so the build can be run ? 


```suggestion
    # DRA_CREDS=$(retry 5 vault kv get -field=data -format=json "${DRA_SECRET_PATH}")
    # export VAULT_ADDR=$(echo ${DRA_CREDS} | jq -r '.vault_addr')
    # export VAULT_ROLE_ID=$(echo ${DRA_CREDS} | jq -r '.vault_role_id')
    # export VAULT_SECRET_ID=$(echo ${DRA_CREDS} | jq -r '.vault_secret_id')
    # TODO just set dummy values until vault secrets can be retrieved.
    DRA_CREDS="fake_path"
    export VAULT_ADDR="vault_address_dummy"
    export VAULT_ROLE_ID="vault_role_id_dummy"
    export VAULT_SECRET_ID="vault_secret_id_dummy"
```